x-pack/dockerlogbeat/docs/install.asciidoc

```diff
@@ -0,0 +1,109 @@
+[[log-driver-installation]]
+== Install and configure the {log-driver}
+
+++++
+<titleabbrev>Install and configure</titleabbrev>
+++++
+
+experimental[]
+
+[float]
+=== Before you begin
+
+Make sure your system meets the following prerequisites:
+
+* Docker: {docker-version}
+* {stack}: Version 7.6.0 and later
+
+//REVIEWERS: Are there any other prereqs?
+//REVIEWERS: Do we want to add a step for setting up a cloud trial? 
+
+[float]
+=== Step 1: Install the {log-driver} plugin
+
+// REVIEWERS: Will the plugin be available on docker hub when 7.6 ships, or will
+// users need to build from source? These instructions assume that the driver
+// will be available at the initial release. If it's not, we can add the steps
+// for building from source.
+
+// Related question: Do we want to document how to build/install from source
+// too?
+
+// The commands shown here are a guess because of course I couldn't test them.
+
+1. Install the plugin from the Docker store:
++
+["source","sh",subs="attributes"]
+----
+docker plugin install store/elastic/elastic-logging-driver:{version} --alias elastic-logging-driver
```

Comment:
Is that `store/` needed?

Reply:
@fearful-symmetry I don't know. I was guessing based on how other plugins are installed. The plugins flagged as "verified publisher" seem to include `store/`.

Reply:
Huh, weird. Yah, I'm...not sure how this part works.

Reply:
OK, I'll leave it as an open issue for now.